src/app/item-list/item-list.component.html

```diff
@@ -1,6 +1,6 @@
-<h2 id="title"> {{capitalizeFirstLetter(this.url[1])}}</h2>
-  <div class="container">
-    <div class="row card-deck">
+<h2 id="title" *ngIf="!selectedStore"> {{capitalizeFirstLetter(this.url[1])}}</h2>
+<h2 id="title" *ngIf="selectedStore!==undefined"> Displaying books of  <em> {{selectedStore.Name}}</em></h2>
```

Comment:
This is not generic and adds extra overhead to `ItemList`. Perhaps moving it to `BooksComponent`'s template would be more appropriate?